lib/util.js

```diff
@@ -154,6 +155,66 @@ export function convert(object, type) {
   }
 }
 
+/**
+ * Create a Data Set like wrapper to seamlessly coerce data types.
+ *
+ * @param rawDS - The Data Set with raw uncoerced data.
+ * @param type - A record assigning a data type to property name.
+ *
+ * @remarks
+ * Use the dispose method prior to throwing a reference to this away. Otherwise
+ * the pipe connecting the two Data Sets will keep the unaccessible coerced
+ * Data Set alive and updated as long as the raw Data Set exists.
+ *
+ * @returns A Data Set like object that saves data into the raw Data Set and
+ * retrieves them from the coerced Data Set.
+ */
+export function typeCoerceDataSet(
```

Comment:
Can you explain better what 's the idea again of this function? I still haven;t grasp the idea of 'type coerced' data sets...

Reply:
I added some more info about the inner workings. It basically just imitates how it worked with the type constructor parameter but without forcing this functionality into the data set itself.